rust/decentraland-godot-lib/src/dcl/js/players.rs

```diff
@@ -0,0 +1,162 @@
+use std::{cell::RefCell, rc::Rc};
+
+use deno_core::{
+    anyhow::{self},
+    error::AnyError,
+    op, Op, OpDecl, OpState,
+};
+
+use crate::dcl::{
+    components::proto_components::common::Color3,
+    crdt::{SceneCrdtState, SceneCrdtStateProtoComponents},
+    scene_apis::{AvatarForUserData, LocalCall, UserData},
+};
+
+pub fn ops() -> Vec<OpDecl> {
+    vec![
+        op_get_player_data::DECL,
+        op_get_connected_players::DECL,
+        op_get_players_in_scene::DECL,
+    ]
+}
+
+#[op]
+async fn op_get_player_data(
+    op_state: Rc<RefCell<OpState>>,
+    user_id: String,
+) -> Result<Option<UserData>, AnyError> {
+    let (sx, rx) = tokio::sync::oneshot::channel::<Option<UserData>>();
+
+    op_state
+        .borrow_mut()
+        .borrow_mut::<Vec<LocalCall>>()
+        .push(LocalCall::PlayersGetPlayerData {
+            user_id,
+            response: sx.into(),
+        });
+
+    rx.await.map_err(|e| anyhow::anyhow!(e))
```

Comment:
```suggestion
    rx.await
        .map_err(|e| anyhow::anyhow!(e))?
        .map_err(|e| anyhow!(e))
```

I'm mapping the error on that way. Just in case the received error is a String.
Up to you to use it or not.